CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+
+Please avoid emailing maintainers found in the MAINTAINERS file directly. They
+are very busy and read the mailing lists.
+
+## Getting Started
+
+- Fork the repository on GitHub
+- Read the [README](README.md) for getting started as a user and learn how/where to ask for help 
+- Continue reading this document for forther development instructions
+- Play with the project, submit bugs, submit pull requests!
+
+## Contribution workflow
+
+This is a rough outline of how to prepare a contribution:
+
+- Create a topic branch from where you want to base your work (usually branched from master).
+- Make commits of logical units.
+- Make sure your commit messages are in the proper format (see below).
+- Push your changes to a topic branch in your fork of the repository.
+- If you changed code:
+   - add automated tests to cover your changes
+- Submit a pull request to the original repository.
+
+## How to build and run the project
+
+```bash
+make
+./scope launch
+```
+
+## How to run the test suite
+
+You can run the linting and unit tests by simply doing
+
+```bash
+make tests
+```
+
+There are integration tests for, but unfortunately it's hard to set them up in fork respositories and that setup is not documented. Help is needed to run integration tests in forks: https://github.com/weaveworks/scope/issues/2192
+
+# Acceptance policy
+
+These things will make a PR more likely to be accepted:
+
+ * a well-described requirement
+ * tests for new code
+ * tests for old code!
+ * new code and tests follow the conventions in old code and tests
+ * a good commit message (see below)
+
+In general, we will merge a PR once two maintainers have endorsed it.
+Trivial changes (e.g., corrections to spelling) may get waved through.
+For substantial changes, more people may become involved, and you might get asked to resubmit the PR or divide the changes into more than one PR.
+
+### Format of the Commit Message
+
+We follow a rough convention for commit messages that is designed to answer two
+questions: what changed and why. The subject line should feature the what and
+the body of the commit should describe the why.
+
+```
+scripts: add the test-cluster command
+
+this uses tmux to setup a test cluster that you can easily kill and
+start for debugging.
+
+Fixes #38
+```
+
+The format can be described more formally as follows:
+
+```
+<subsystem>: <what changed>
+<BLANK LINE>
+<why this change was made>
+<BLANK LINE>
+<footer>
+```
+
+The first line is the subject and should be no longer than 70 characters, the
+second line is always blank, and other lines should be wrapped at 80 characters.
+This allows the message to be easier to read on GitHub as well as in various
+git tools.
+
+## 3rd party plugins
+So you've built a Scope plugin. Where should it live?
+
+Until it matures, it should live in your own repo.
```

Comment:
Do we have more concrete acceptance criteria?

Maybe it'd be generally a good idea to invite people to speak about their scope plugin on the mailing list to nurture collaboration?

Reply:
Yes, or present it on a weekly meeting. I will do that.

Reply:
Maybe mention this here as well?

> Until it matures, it should live in your own repo. We love hearing of your work, so please mention your work on the mailing list and/or our community meeting. You might find new contributors this way as well.